brianlong/www.validators.app
# 184781969 sort versions by stake

#### What's this PR do?
- sort software versions by stake

#### How should this be manually tested?
- run tests
- check if software versions on home page are sorted by stake percent correctly

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184781969)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
